inc/commoninjectionlib.class.php

```diff
@@ -606,14 +606,56 @@ private function getFieldValue(
                 $item    = new $tmptype();
                 if ($item instanceof CommonTreeDropdown) {
                    // use findID instead of getID
-                    $input =  ['completename' => $value,
+                    $input =  [
+                        'completename' => $value,
                         'entities_id'  => $this->entity
                     ];
 
-                    if ($item->canCreate() && $this->rights['add_dropdown']) {
-                        $id = $item->import($input);
+                    if ($item->getType() == 'Entity') {
+                        $entity = new Entity();
+                        $sons = getSonsOf('glpi_entities', $input['entities_id']);
+                        if (strpos($value, '>')) {
```

Comment:
Technically, a name can't have a ‘>’?
Perhaps it would be easier to search if `name = $value OR completename = $value`?

Reply:
Yes, I agree, but I mean that the (simple) name of an entity can contain a ‘>’, it's not blocked.
![image](https://github.com/user-attachments/assets/a021cafe-9f58-4a55-88c1-6dd0d8fb5bb6)